bundle-workflow/src/test_workflow/integ_test/local_test_cluster.py

```diff
@@ -84,6 +93,14 @@ def disable_security(self, dir):
             shell=True,
         )
 
+    def __add_plugin_specific_config(self, additional_config: dict, file):
```

Comment:
these extra opensearch settings can be configuration dependent as well going forward, e.g. some settings applicable only for with-security config? I think it would be better if we have this logic outside of `local_test_cluster` and have the test_suite provide these additional settings for spinning up the cluster.

Reply:
How would you structure this in is test-manifest?

Reply:
Offline conversation: Added the `additional-cluster-configs` key under `integ-test`